earthly/rust/Earthfile

```diff
@@ -82,7 +82,7 @@ rust-base:
     # Ensure we have all the necessary targets
     RUN rustup target add wasm32-unknown-unknown
     RUN rustup target add wasm32-wasip1
-    # RUN rustup target add wasm32-wasip2 # wasm32-wasip2 not yet available in stable - Try again in 1.82
+    RUN rustup target add wasm32-wasip2 # wasm32-wasip2 not yet available in stable - Try again in 1.82
```

Comment:
Why uncomment this line ?
```suggestion
    RUN rustup target add wasm32-wasip2 # wasm32-wasip2 not yet available in stable - Try again in 1.84
```

Reply:
Because I want that target and it seems like its available now.

Reply:
`wasm32-wasip2` still experimental no ?
https://doc.rust-lang.org/nightly/rustc/platform-support/wasm32-wasip2.html